client/src/pages/Country/Country.tsx

```diff
@@ -84,6 +85,39 @@ export const Country: React.FC<CountryProps> = ({ country }) => {
             </div>
           ))}
         </div>
+        <h3>Publication Date vs Time to Trend</h3>
```

Comment:
NAB, can merge without: Can also remove the original 
`   {genResults.map((result, index) => (
            <div key={index}>
              <p>{result?.title}</p>
              <hr style={{ width: "30%" }} />
            </div>
          ))}` 

Since we no longer use the video list

Reply:
Done.